farming_assistant/lib/models/enums/location_type.dart

```diff
@@ -0,0 +1,6 @@
+enum LocationType {
```

Comment:
For your knowledge astea o sa vina nitel schimbate dupa ce vin eu cu harta

Reply:
Doamne ajuta